hera_qm/xrfi.py

```diff
@@ -1696,6 +1705,7 @@ def xrfi_run_step(uv_files=None, uv=None, uvf_apriori=None,
 
 def xrfi_run(ocalfits_files=None, acalfits_files=None, model_files=None,
              data_files=None, a_priori_flag_yaml=None,
+             a_priori_ants_only=False,
```

Comment:
Again, I'd suggest default True

Reply:
These need to have default = False to preserve many of the unittests. The script interace defaults everything to True though.

Reply:
I'm fine with this.